.kokoro/tests/run_single_test.sh

```diff
@@ -41,6 +41,32 @@ fi
 nox -s "$RUN_TESTS_SESSION"
 EXIT=$?
 
+# Inject region tag data into the test log
+if [[ "${INJECT_REGION_TAGS:-}" == "true" ]]; then
+    export REGION_TAG_PARSER_DIR="/tmp/region-tag-parser"
+    export PARSER_PATH="${REGION_TAG_PARSER_DIR}/wizard-py/cli.py"
+    export PIP_PATH=/tmp/pyyaml
+    export PYTHONPATH="$PYTHONPATH:$PIP_PATH"
+
+    export XUNIT_PATH="$PWD/sponge_log.xml"
+    export XUNIT_TMP_PATH="${HOME}/drift_tmp_$(uuidgen).xml"
+
+    if [[ -f "$XUNIT_PATH" ]]; then
+        echo "=== Injecting region tags into XUnit output ==="
+        echo "Processing XUnit output file: $XUNIT_PATH (saving output to $XUNIT_TMP_PATH)"
+
+        cat "$XUNIT_PATH" | python3.7 "$PARSER_PATH" inject-snippet-mapping --output_file "$XUNIT_TMP_PATH" "$PWD"
+        if [[ $? -eq 0 ]]; then
+            mv $XUNIT_TMP_PATH $XUNIT_PATH
```

Comment:
Are you 100% sure that there is a file when the script exit with 0?
If not, please check the file existance.

Also, does it make sense to log something here as well?

Reply:
If the file at `$XUNIT_TMP_PATH` doesn't exist, then the `mv` command should fail.

IMO, logging something here would be overkill. 🙂

Reply:
So if `mv` fails, the test is marked as failed even if just the injection here failed?
I don't like it.

Reply:
Well, actually, there's no `-e` flag set in this file. So it may keep running.
I still like the explicit check, but I think I can live with the current code.